### ISSUE TITLE
The more actions button in chats is labeled "go back"

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2699]
**Describe the bug**
The button just one tab before the message list or the pins is labeled "go back", even though it opens an actions menu.

**To Reproduce**
1. Press ctrl+windows+enter to activate narrator (you deactivate it with the same keystroke).
2. Opnen a chat in Unigram.
3. Press shift tab some times, just until you reach the button which opens a menu (with the search and silence options).
4. Hear that the button is labeled "go back".

**Expected behavior**
The button should be labeled more appropriately, maybe as "more actions" or "more" (at least one of these labels is used by Unigram already).

**Version Info**
 - Unigram: Unigram 8.6 (7292) X64 Direct
 - Windows: Versión 21H2 (compilación de SO 19044.1586)

**Comment by FrayxRulez** [comment]
Fixed in https://github.com/UnigramDev/Unigram/commit/9d3794d83358525fb45d346dfc6af50da1adb65f
